controllers/controller_ref_manager.go

```diff
@@ -288,6 +289,97 @@ func (m *EtcdDruidRefManager) ClaimCronJob(ctx context.Context, cronJob *batchv1
 	return claimed, utilerrors.NewAggregate(errlist)
 }
 
+func (m *EtcdDruidRefManager) ClaimServiceAccount(ctx context.Context, sa *corev1.ServiceAccount, filters ...func(*corev1.ServiceAccount) bool) (*corev1.ServiceAccount, error) {
```

Comment:
Why do we need to claim the `ServiceAccount` here? Isn't it sufficient to just create it via the chart? Same goes for `Role` and `RoleBinding`.

Reply:
@timuthy you're right. @aaronfern I think this requires some history of why the claim logic was even necessary in the first place. Initially (3 years ago), there was only the etcd statefulset (that contains the etcd container and backup-restore sidecar) which was directly deployed by gardener. Later, we realised the need for a controller to perform tasks that a sidecar can't, and hence built etcd-druid. Druid could create new etcds with no issues, but from a backward-compatibility perspective, landscapes were already running multiple etcd statefulsets, so druid now had to "claim" these.

For the new resources you're introducing here (serviceaccount, role, rolebinding for the backup sidecar), you need not claim them, as you're sure that these resources are created fresh by druid.